### PR TITLE
New description for marketplace

### PR DESCRIPTION
**Pull request by sigewuzhere**
### Describe the purpose of your pull request

New description for marketplace

### Requirement checklist (only if applicable)

- [ ] I have covered the applied changes with automated tests.
- [ ] I have executed the full automated test set against my changes.
- [ ] I have validated my changes against all supported platform versions.
- [x] I have read and accepted the [contribution agreement](https://github.com/configcat/legal/blob/main/contribution-agreement.md).

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+
   ### Configuring the Step
 
-  * Create a new [ConfigCat Management API credential](https://app.configcat.com/my-account/public-api-credentials) and store its values in secret
-  environment variables with the following names: `CONFIGCAT_API_USER`, `CONFIGCAT_API_PASS`.
-  * [Get the ID of your ConfigCat Config](https://configcat.com/docs/advanced/code-references/overview#config-id) that you want to associate with your repository
-  and use it as the `config_id` input. The scanner will use this ID to determine which feature flags and settings to search for in your source code.
+  1. Log in to [ConfigCat](https://app.configcat.com/)
+  1. Create new [ConfigCat Management API credentials](https://app.configcat.com/my-account/public-api-credentials/) 
+  1. Store your credentials in environment variables under the following names: `CONFIGCAT_API_USER`, `CONFIGCAT_API_PASS`.
```
Comment (zoltan-david) [inline step.yml:14]: I'm missing the context. Where (which system/machine/etc) should I store these variables?

File: step.yml [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+  1. [Get the ID of your ConfigCat config](https://configcat.com/docs/advanced/code-references/overview#config-id) that you want to associate your repository with and add it as the `config_id` input. The Step will use the config ID to determine which feature flags to search for in the source code.
```
Comment (zoltan-david) [inline step.yml:15]: Add your [ConfigCat config ID](https://configcat.com/docs/advanced/code-references/overview#config-id) to the `config_id` input of this script. The config ID tells the script which feature flags it should search for in your source code.

File: step.yml [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+  ConfigCat Feature Flags
 summary: |
-  Scan ConfigCat feature flag and setting usages in your source code and uploads the found code references to [ConfigCat](https://configcat.com).
+  Scans the source code for feature flags and uploads the references to the [ConfigCat Dashboard](https://app.configcat.com/).
 description: |-
-  Scan ConfigCat feature flag and setting usages in your source code and uploads the found code references to [ConfigCat](https://configcat.com).  
-  This feature makes the erasement of technical debt easier, as it can show which repositories reference your
-  feature flags and settings in one centralized place on your [Dashboard](https://app.configcat.com/).  
-  
+  This Step helps you get rid of technical debt by scanning the source code and highlighting the feature flag usages for each feature flag on the the [ConfigCat Dashboard](https://app.configcat.com/).
```
Comment (kp-cat) [inline step.yml:6]:
```suggestion
  This Step helps you get rid of technical debt by scanning the source code and highlighting the feature flag usages for each feature flag on the [ConfigCat Dashboard](https://app.configcat.com/).
```

File: step.yml (hunk 1 above)
Comment (kp-cat) [inline step.yml:14]:
```suggestion
  1. Store your credentials in secret environment variables under the following names: `CONFIGCAT_API_USER`, `CONFIGCAT_API_PASS`.
```

File: step.yml [hunk 4]
```diff
@@ -1,24 +1,24 @@
 title: |-
-  ConfigCat Scan Repository
+  ConfigCat Feature Flags
```
Comment (zoltan-david) [inline step.yml:2]:
```suggestion
  ConfigCat Feature Flag Sync
```

File: step.yml [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
+
   ### Configuring the Step
 
-  * Create a new [ConfigCat Management API credential](https://app.configcat.com/my-account/public-api-credentials) and store its values in secret
-  environment variables with the following names: `CONFIGCAT_API_USER`, `CONFIGCAT_API_PASS`.
-  * [Get the ID of your ConfigCat Config](https://configcat.com/docs/advanced/code-references/overview#config-id) that you want to associate with your repository
-  and use it as the `config_id` input. The scanner will use this ID to determine which feature flags and settings to search for in your source code.
+  1. Log in to [ConfigCat](https://app.configcat.com/)
+  1. Create new [ConfigCat Management API credentials](https://app.configcat.com/my-account/public-api-credentials/) 
+  1. Store your credentials in secret environment variables under the following names: `CONFIGCAT_API_USER`, `CONFIGCAT_API_PASS`.
```
Comment (zoltan-david) [inline step.yml:14]:
```suggestion
  1. Store your credentials in the following Bitrise secret environment variables: `CONFIGCAT_API_USER`, `CONFIGCAT_API_PASS`.
```

File: step.yml [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
+  1. [Get the ID of your ConfigCat config](https://configcat.com/docs/advanced/code-references/overview#config-id) that you want to associate your repository with and add it as the `config_id` input. The Step will use the config ID to determine which feature flags to search for in the source code.
```
Comment (zoltan-david) [inline step.yml:15]:
```suggestion
  1. Add your [ConfigCat config ID](https://configcat.com/docs/advanced/code-references/overview#config-id) to the `ID of the ConfigCat config to scan against` input variable of this step (`config_id` in bitrise.yml). The config ID tells the script which feature flags it should search for in your source code.```

**Pull request merged**